PowerPointLabs/PowerPointLabs/CropToShape.cs

```diff
@@ -228,7 +235,7 @@ private static void TakeScreenshotProxy(PowerPoint.ShapeRange shapeRange)
             shapeRange.Visible = Office.MsoTriState.msoTrue;
         }
 
-        public static PowerPoint.ShapeRange UngroupAllForShapeRange(PowerPoint.ShapeRange range, bool remove = true)
+        public static PowerPoint.ShapeRange UngroupAllForShapeRange(PowerPoint.ShapeRange range, bool remove = false)

```

Comment:
looks like param `remove` is never used.
1. can you help to make this method `private` & remove this unused param.
2. remove its related logic and unused method